release-handbook/pdf_generation_guide.md

```diff
@@ -55,7 +55,7 @@ Instructions for each case are listed in the sections below.
 4. Paste the following command, updating the version and patch numbers as necessary:
 
     ```
-    npx docs-to-pdf --initialDocURLs="https://docs.zowe.org/v1.28.x/getting-started/overview,https://docs.zowe.org/v1.28.x/user-guide/install-overview,https://docs.zowe.org/v1.28.x/user-guide/zowe-getting-started-tutorial,https://docs.zowe.org/v1.28.x/extend/extend-zowe-overview,https://docs.zowe.org/v1.28.x/troubleshoot/troubleshooting,https://docs.zowe.org/v1.28.x/contribute/roadmap-contribute,https://docs.zowe.org/v1.28.x/appendix/zowe-cli-command-reference" --cssStyle=":root{--ifm-font-size-base: 80%;}" --contentSelector="article" --paginationSelector=".pagination-nav__link--next" --excludeSelectors=".no-pdf,.announcementBar_UUUQ,nav.navbar,.docSidebarContainer_3pwe,.docMainContainer_2pgU .col.col--3,footer.footer,.docItemContainer_2szM > .margin-vert--xl > .row > .col:first-child,.pagination-nav,.docsRating,.navbar__inner,.thin-scrollbar" --coverImage="https://docs.zowe.org/img/zowe-icon.png" --coverTitle="Zowe Version 1.28.x Documentation" --outputPDFFilename="static/zowe-docs-v1.28.x.pdf" --protocolTimeout=1800000 
+    npx docs-to-pdf --initialDocURLs="https://docs.zowe.org/v1.28.x/whats-new/zowe-announcements,https://docs.zowe.org/v1.28.x/getting-started/overview,https://docs.zowe.org/v1.28.x/user-guide/install-overview,https://docs.zowe.org/v1.28.x/user-guide/zowe-getting-started-tutorial,https://docs.zowe.org/v1.28.x/extend/extend-zowe-overview,https://docs.zowe.org/v1.28.x/troubleshoot/troubleshooting,https://docs.zowe.org/v1.28.x/contribute/roadmap-contribute,https://docs.zowe.org/v1.28.x/appendix/zowe-cli-command-reference" --cssStyle=":root{--ifm-font-size-base: 80%;}" --contentSelector="article" --paginationSelector=".pagination-nav__link--next" --excludeSelectors=".no-pdf,.announcementBar_UUUQ,nav.navbar,.docSidebarContainer_3pwe,.docMainContainer_2pgU .col.col--3,footer.footer,.docItemContainer_2szM > .margin-vert--xl > .row > .col:first-child,.pagination-nav,.docsRating,.navbar__inner,.thin-scrollbar" --coverImage="https://docs.zowe.org/img/zowe-icon.png" --coverTitle="Zowe Version 1.28.x Documentation" --outputPDFFilename="static/zowe-docs-v1.28.x.pdf" --protocolTimeout=1800000 
```

Comment:
The added link https://docs.zowe.org/v1.28.x/whats-new/zowe-announcements doesn't seem to work for me

Reply:
This doesn't really matter as V1 support is nearly ending, and the command will work for previous V2 releases.